pyo3-build-config/src/impl_.rs

```diff
@@ -864,7 +952,7 @@ fn ends_with(entry: &DirEntry, pat: &str) -> bool {
 /// ```
 ///
 /// [1]: https://github.com/python/cpython/blob/3.5/Lib/sysconfig.py#L389
-fn find_sysconfigdata(cross: &CrossCompileConfig) -> Result<PathBuf> {
+pub fn find_sysconfigdata(cross: &CrossCompileConfig) -> Result<PathBuf> {
```

Comment:
I would probably refactor this function a bit further, so that the `find_sysconfigdata` function which we share with `maturin` just returns sorted / deduplicated / canonicalized `Vec<PathBuf>`. 

The error messages for empty or too many results are pretty specific to the pyo3 build script.